src/solver/layernorm/forward_layernorm4d_ck.cpp

```diff
@@ -235,7 +235,7 @@ bool Layernorm4DCKForward::IsApplicable(
         return false;
     if(!problem.IsLargeSize())
         return false;
-    if(!ck_utility::is_ck_supported_hardware(context.GetStream()))
```

Comment:
`is_ck_supported_hardware` relates to the `Legacy CK`, while this solver uses `CK`.

Reply:
See https://github.com/ROCm/MIOpen/pull/3749#discussion_r2109613197